apps/designer/app/designer/features/style-panel/controls/font-weight/is-supported-font-weight.test.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, test } from "@jest/globals";
+import { FontAsset } from "@webstudio-is/asset-uploader";
+import { nanoid } from "nanoid";
+import { isSupportedFontWeight } from "./is-supported-font-weight";
+
+const createServerAsset = (meta: FontAsset["meta"]): FontAsset => ({
+  id: nanoid(),
+  type: "font" as const,
```

Comment:
as const not necessary with return type

Reply:
correct, was fixed by a type and forgotten